src/tests/system/tests/test_sss_ssh_knownhosts.py

```diff
@@ -0,0 +1,122 @@
+"""
+sss_ssh_knownhosts tests.
```

Comment:
Can we name the file, test_ssh.py?

And make the test name a little more readable? Like

test_ssh__checking_sss_ssh_knownhosts_can_find_keys_by_name

Reply:
How does it improve things?

This does not test ssh at all, it just tests the sss_ssh_knownhosts tool. We have test_sss_cache.py, test_sss_override.py, test_sssctl.py so test_sss_ssh_knownhosts.py sounds like a good choice to place tests for sss_ssh_knonwhosts tool.